.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npm run lint
```

Comment:
Let's remove the husky. We can add the lint rule to our CI pipeline and fail the CI if the lint fails